### DISCUSSION TITLE
Custom border radius for pages

### DISCUSSION THREAD

**Discussion by Lovasz-Akos** [Floorp-Projects/Floorp#676] [Feature Request]
I'd love the ability to make the corner radius under "Settings > Design > Round the corners of pages" a custom value, to make the corner smaller or larger depending on prefrence

**Comment by TheChilledBuffalo** [comment]
Yeah that would be a good idea 👀 

Maybe we'll have to set a limit for it as well, so that people wouldn't overdo it.

**Comment by Lovasz-Akos** [comment]
would be funny if the radius was so big that the pages are a circle lol
